orator/orm/builder.py

```diff
@@ -286,7 +286,7 @@ def lists(self, column, key=None):
 
                 results[i] = self._model.new_from_builder(fill).column
 
-    def paginate(self, per_page=None, current_page=None, columns=None):
+    def paginate(self, per_page=None, current_page=None, column_count=None, columns=None):
```

Comment:
Since `column_count` has a default value, can you move this as a last param ? That way, existing code won't break if caller passes only 3 arguments, thinking 3rd one is `column`